firmware/hw_layer/ports/stm32/stm32f7/mpu_util.cpp

```diff
@@ -209,7 +221,8 @@ void stm32_standby() {
 
 	// Wait for event - this should never return as it kills the chip until a reset
 	__WFE();
+	__WFE();
```

Comment:
do we now need __WFE _twice_? your PR adds second time?

Reply:
Funny enough, yes and no. 
So if we are using WFE then yes, needs twice. Its an oddity: https://devzone.nordicsemi.com/f/nordic-q-a/10932/can-someone-explain-to-me-how-nordic-nrf51822-sleep-mode-works

However, with WFI we do not need it. Standby is not used right now at all, and may be removed altogether. I am working on testing it now. Hence why I didn't much care for the standby function.

Reply:
looks like this code comment with magic link has to be in the code then

Reply:
Yeah, but since we are waking on interrupt instead of Event, might as well forget about it. especially since I couldnt get Event to work anyway, (at least for wakeup). WFI is more direct, and tested to be working.